packages/grid/_modules_/grid/models/params/gridColumnResizeParams.ts

```diff
@@ -0,0 +1,17 @@
+/**
+ * Object passed as parameter of the column resize event.
+ */
+export interface GridColumnResizeParams {
+  /**
+   * The HTMLElement column header element.
+   */
+  element?: HTMLElement | null;
```

Comment:
Why do we care about the element? For instance, I was expecting the new width to be a lot more interesting than this.

Reply:
It isn't. I saw that other params pass it. Also, it already is available in the `colDef`.

Reply:
I think we should add the width in the params as it's the most important value of the event.

Reply:
added